cmd/secret/cmd/root_test.go

```diff
@@ -0,0 +1,118 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestRoot_Set(t *testing.T) {
+	t.Run("expect one keys", func(t *testing.T) {
+		key := "key value"
+		path := "testFile.txt"
+		version := "test"
+		r := New(version)
+		r.cmd.SetArgs([]string{"set", "--key", key, "--value", "test value", "--cipherKey", "ck", "--path", path})
+		err := r.Execute(context.Background())
```

Comment:
Try to reuse context, may use `WithTimeout` constructor to restrict commands execution by some timeframe.